Sources_v3/Controllers/UserController/UserController_Tests.swift

```diff
@@ -28,6 +28,8 @@ final class UserController_Tests: StressTestCase {
     }
     
     override func tearDown() {
+        env.userUpdater?.cleanUp()
+        
```

Comment:
Why do we need this? The whole `env` is destroyed on L39

Reply:
We need this because controller owns the updater and updater holds a strong reference to the controller by a completion closure. Even though we release the env, we have a retain cycle so controller and updater cannot be released.

Exactly the same we have for all `XXXUpdater` types. We do `apiClient.cleanUp` as the first step in `tearDown`